src/mlpack/core/data/split_data.hpp

```diff
@@ -169,6 +142,9 @@ void StratifiedSplit(const arma::Mat<T>& input,
  *                testData, trainLabel, testLabel, 0.3);
  * @endcode
  *
+ * @tparam T Type of the elements of the input matrix.
+ * @tparam LabelsType Type of input labels. It can be arma::Mat, arma::row,
```

Comment:
```suggestion
 * @tparam LabelsType Type of input labels. It can be arma::Mat, arma::Row,
```
Can it be `arma::Col` too?

Reply:
I don't think it can be because of lines like these
```
trainLabel(0, i) = inputLabel(0, order(i));
testLabel(0, i - trainSize) = inputLabel(0, order(i));
```
here we are accessing using (0, col_num). This thing will probably wont work for `arma:Col`.

Reply:
Not sure, I see the issue, it should work for `arma::col` as well.

Reply:
You are right @zoq. Will change the docs to reflect this.

Reply:
In fact, if you are always accessing the first row, you can just do, e.g.,

```
trainLabel(i) = inputLabel(order(i));
testLabel(i - trainSize) = inputLabel(order(i));
```